### PR TITLE
batches: test the fix in #666

### PR DESCRIPTION
**Pull request by LawnGnome**
@mrnugget suggested this test, and quite right he was, too.

### CODE REVIEW

File: internal/batches/ui/task_exec_tui_test.go [hunk 1]
```diff
@@ -205,6 +197,70 @@ func TestTaskExecTUI_Integration(t *testing.T) {
 	})
 }
 
+func TestProgressUpdateAfterComplete(t *testing.T) {
+	if runtime.GOOS == "windows" {
+		t.Skip("Something emits different escape codes on windows.")
```
Comment (eseliger) [inline internal/batches/ui/task_exec_tui_test.go:202]: Of course 😆

**Review by eseliger — APPROVED** [review]

File: internal/batches/ui/task_exec_tui_test.go (hunk 1 above)
Comment (LawnGnome) [reply internal/batches/ui/task_exec_tui_test.go:202]: In fairness, I copy/pasted that from the other integration test — but only after it failed on Windows. :window:

File: internal/batches/ui/task_exec_tui_test.go [hunk 2]
```diff
@@ -205,6 +197,70 @@ func TestTaskExecTUI_Integration(t *testing.T) {
...
+		"├── github.com/sourcegraph/sourcegraph  echo Hello World > README.md          0s",
+		"└── github.com/sourcegraph/src-cli      Downloading archive                   0s",
+		"",
+	})
+
+	// Now mark the progress as complete.
+	printer.progress.Complete()
+
+	// Now send another update. This would panic before the relevant fix was
+	// merged in #666.
```
Comment (mrnugget) [inline internal/batches/ui/task_exec_tui_test.go:250]: I just realised we forgot to celebrate #666, the PR of the beast, accordingly (by doing a little ritual in the comments or something) :(

**Review by mrnugget — APPROVED** [review]

File: internal/batches/ui/task_exec_tui_test.go (hunk 2 above)
Comment (LawnGnome) [reply internal/batches/ui/task_exec_tui_test.go:250]: From the current changelog:

image

Comment (mrnugget) [reply internal/batches/ui/task_exec_tui_test.go:250]: Nice, that's the spirit (we summoned)

**Pull request merged**